src/cc/lib/graph/metadata.h

```diff
@@ -19,7 +19,7 @@ struct Metadata
 {
     Metadata() = default;
     explicit Metadata(std::filesystem::path path, std::string config_path = "",
-                      std::shared_ptr<Logger> logger = nullptr);
+                      std::shared_ptr<Logger> logger = nullptr, bool skip_feature_loading = false);
```

Comment:
Lets put this parameter before logger, that way the "nullptr, skip_feature_loading" in py_server.cc can be replaced with just "skip_feature_loading"

Reply:
Sure, just be aware that this will break the backward compatibility if anyone is using this module outside of DeepGNN.